docs/getting-started-guides/ubuntu/rancher.md

```diff
@@ -0,0 +1,198 @@
+---
+title: Rancher Integration with Ubuntu Kubernetes
+---
+
+{% capture overview %}
+This repository explains how to deploy Rancher 2.0alpha on Canonical Kubernetes. 
+
+These steps are currently in alpha/testing phase and will most likely change. 
+
+The original documentation for this integration can be found here [https://github.com/CalvinHartwell/canonical-kubernetes-rancher/](https://github.com/CalvinHartwell/canonical-kubernetes-rancher/). 
+
+{% endcapture %}
+{% capture prerequisites %}
+This page assumes you have a working Juju deployed Canonical Kubernetes Cluster. 
+{% endcapture %}
+
+
+{% capture steps %}
+## Implementation
+
+To deploy Rancher, we just need to run the Rancher container workload on-top of Kubernetes. Rancher provides their containers through dockerhub ([https://hub.docker.com/r/rancher/server/tags/](https://hub.docker.com/r/rancher/server/tags/)) and can be downloaded freely from the internet. If you're running your own registry or have an offline deployment, the container should be downloaded and pushed to the private registry.  
```

Comment:
Wording suggestion to make this more direct:
> You may deploy Rancher on top of Kubernetes. 

In the last sentence in the paragraph, you reference "the private registry" as though it's been spoken of already, but i didn't see it referenced. Would better phrasing be "a private registry."

Reply:
Fixed.